README.md

```diff
@@ -1,2 +1,58 @@
 # jupyter-js-output-area
 Javascript APIs for Jupyter output areas
+
+## Demo
+See [./demo/demo.html](demo/demo.html).
+
+## Install
+### Stable (npm):
+```
+npm install jupyter-js-output-area
+```
+
+### Dev install
+```
+git clone https://github.com/jupyter/jupyter-js-output-area.git
+cd jupyter-js-output-area
+```
+
+To build
+```
+npm install --dev
```

Comment:
If they've cloned it, they don't have to run `npm install --dev`, they can just run

```
npm install
```